cfme/tests/containers/test_relationships_tables.py

```diff
@@ -33,17 +30,29 @@
                           'Containers',
                           'Node'])
 def test_pods_rel(provider, rel):
+    """   This module verifies the integrity of the Relationships table
+          We also verify that clicking on the Relationships table field
+          takes the user to the correct page, and the number of rows
+          that appears on that page is equal to the number in the
+          Relationships table
+          nginx pod is excluded because it lacks replicators, metrics pod is excluded
+          because after the metrics are deployed, it just lingers around, it also
+          does not have any services and replicators associated with it
+    """
```

Comment:
Just for the record, I don't believe this is handled correctly but it will not block this PR from getting merged.